doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -277,3 +278,105 @@ Set used by the SPM sample
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52 DK (emulating nRF52810) <ug_nrf52>`   |PCA10040     |:ref:`nrf52dk_nrf52810 <nrf52dk_nrf52810>`      |``nrf52dk_nrf52810``            |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+
+.. set20_end
+
+Set used by BLE samples (Central HIDS, Central UART, Mesh Light, Peripheral HIDS Keyboard, Peripheral HIDS mouse, Peripheral UART, NUS Shell Transport)
+
+.. set21_start
+
+
+
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|Hardware platforms                                |PCA          |Board name                                      |Build target                    |
++==================================================+=============+================================================+================================+
+|:ref:`nRF5340 PDK <ug_nrf5340>`                   |PCA10095     |:ref:`nrf5340pdk_nrf5340 <nrf5340pdk_nrf5340>`  |``nrf5340pdk_nrf5340_cpuapp``   |
+|                                                  |             |                                                |                                |
+|                                                  |             |                                                |``nrf5340pdk_nrf5340_cpuappns`` |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52840 DK <ug_nrf52>`                     |PCA10056     |:ref:`nrf52840dk_nrf52840 <nrf52840dk_nrf52840>`|``nrf52840dk_nrf52840``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52833 DK <ug_nrf52>`                     |PCA10100     |:ref:`nrf52833dk_nrf52833 <nrf52833dk_nrf52833>`|``nrf52833dk_nrf52833``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52833 DK (emulating nRF52820) <ug_nrf52>`|PCA10100     |:ref:`nrf52833dk_nrf52820 <nrf52833dk_nrf52820>`|``nrf52833dk_nrf52820``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52 DK <ug_nrf52>`                        |PCA10040     |:ref:`nrf52dk_nrf52832 <nrf52dk_nrf52832>`      |``nrf52dk_nrf52832``            |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+
+.. set21_end
+
+
+Set used by BLE sample (Peripheral GATT Discovery Manager)
```

Comment:
BLE -> Bluetooth LE

Why do we have a different group for this one?

Reply:
Added a new group as nRF5340 PDK is not there in the supported boards in this sample (comparing this group with set20).